vm/rust/src/jsonrpc.rs

```diff
@@ -203,18 +244,15 @@ pub struct FunctionInvocation {
     pub calls: Vec<FunctionInvocation>,
     pub events: Vec<OrderedEvent>,
     pub messages: Vec<OrderedMessage>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub execution_resources: Option<ExecutionResources>,
```

Comment:
We are missing this when we adapt traces from the feeder. Do you mind adding that on top of this PR @joshklop?

Reply:
Will do.